src/Common/src/CoreLib/System/Runtime/CompilerServices/ValueTaskAwaiter.cs

```diff
@@ -7,7 +7,7 @@
 using System.Threading.Tasks;
 using System.Threading.Tasks.Sources;
 
-#if !netstandard
+#if !NETSTANDARD2_0
```

Comment:
I think some of these ifdefs, like this one, are stale.  We no longer build a netstandard package that includes this file.

Reply:
We have a lot of dead conditions in our libraries. Do we have a tool that can remove those?

Reply:
What would the tool do?

Reply:
Check for dead code by building on all configurations. @danmosemsft I think you wrote something similar for coreclr at some point?

Reply:
I think @ericstj knows how to do it using AsmDiff and the ILLinker which strips out unused code.

Reply:
This is a preprocessor directive which wouldn't show up in IL.